gstreamer/src/query.rs

```diff
@@ -208,17 +114,182 @@ impl fmt::Debug for QueryRef {
     }
 }
 
-impl ToOwned for QueryRef {
-    type Owned = GstRc<QueryRef>;
+unsafe impl MiniObject for QueryRef {
```

Comment:
Can you move these things above `impl QueryRef` again to keep the diff more readable?

Reply:
Sure!